source/client/process_impl.cc

```diff
@@ -23,6 +24,11 @@
 #include "external/envoy/source/common/runtime/runtime_impl.h"
 #include "external/envoy/source/common/singleton/manager_impl.h"
 #include "external/envoy/source/common/thread_local/thread_local_impl.h"
+#include "external/envoy/source/extensions/tracers/well_known_names.h"
+
+#ifndef zipkin_disabled
```

Comment:
In general, we try and avoid `#ifdef` in Envoy, maybe take a `TODO` to use some kind of static module registration (as done by Envoy) when we have more than null/Zipkin tracers?

Reply:
Added a TODO for that.